wagtail/admin/templates/wagtailadmin/pages/confirm_delete.html

```diff
@@ -68,6 +68,11 @@
 
         <form action="{% url 'wagtailadmin_pages:delete' page.id %}" method="POST">
             {% csrf_token %}
+            {% if confirm_before_delete %}
+                <input type="checkbox" required>
```

Comment:
Thanks for your review. Actually, I had made this checkbox required so that users should not be able to submit the form without clicking it.

Reply:
Ah, I see! Usually I'd expect form validation to be enforced on the server side, since it's possible for a malicious user to bypass client-side checks, and so I overlooked the `required` attribute... but in this case the code isn't doing anything "security-critical", it's just a UI element to slow the user down. So you're right - a client-side check is fine here.